ehitchhiking-ui/src/app/shared/components/active-trips-modal/active-trips-modal.component.html

```diff
@@ -0,0 +1,50 @@
+<div class="modal-wrapper trips-modal-container">
+  <app-pre-loading [isLoading]="loading" [size]="loaderSize"></app-pre-loading>
+  <header class="modal-header form-row">
+    <span mat-dialog-title class="trips-title">Active trips</span>
+    <i class="material-icons icon-size">
+      history
```

Comment:
It is not a good idea to show one icon in two modals. Pls remove or and the other icon

Reply:
removed it